### PR TITLE
fix: tags is too compact up and down

### PR DESCRIPTION
**Pull request by c0dedance**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

#38463 
image

### 📝 Changelog

旧文档中对tag添加的边距在dumi中没有生效/缺失?
image

after
image

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | add tag margin that not too compact|
| 🇨🇳 Chinese | 给tag添加margin不至于太紧凑 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/common/GlobalStyles.tsx [hunk 1]
```diff
@@ -840,6 +840,9 @@ const GlobalStyles = () => {
             &,
             .code-box-demo {
               background-color: ${token.colorBgContainer};
+              .ant-tag {
+                  margin-bottom: 8px;
+              }
```
Comment (afc163) [inline .dumi/theme/common/GlobalStyles.tsx:845]: 不应该加在 demo 的全局样式里。

**Comment by MadCcc** [comment]
用 Space 组件改一下这个 Demo 吧

**Comment by c0dedance** [comment]
> 用 Space 组件改一下这个 Demo 吧

ok~

File: .dumi/theme/common/GlobalStyles.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ const GlobalStyles = () => {
           ol {
             list-style: none;
           }
-
+
```
Comment (afc163) [inline .dumi/theme/common/GlobalStyles.tsx:64]:
```suggestion

```

Comment (afc163) [reply .dumi/theme/common/GlobalStyles.tsx:64]: 不应该有空格

File: components/tag/demo/colorful.tsx [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { Divider, Tag } from 'antd';
+import { Divider, Space, Tag } from 'antd';
 
 const App: React.FC = () => (
   <>
     <Divider orientation="left">Presets</Divider>
-    <div>
+    <Space wrap value={[8,0]}>
```
[the remark this answers is not in the document]
Comment (c0dedance) [reply components/tag/demo/colorful.tsx]: Oops,my low-level mistake

**Comment by afc163** [comment]
图片
fixed

**Pull request closed without merging**